### PR TITLE
register javascript update 1

### PR DESCRIPTION
**Pull request by shu1513**
trying to use pull request. 

Instead of using oninput and onsubmit with html, I used addEventListener and "DOMContentLoaded" in the javascript

### CODE REVIEW

File: static/register_script.js
Comment (shu1513) [inline static/register_script.js:1]: deleted unused codes.

**Review by nathanieltsmith — CHANGES_REQUESTED** [review]
You probably want to add the binary files to .gitignore file and remove them from the PR. so they aren't kept in version control

File: templates/register.html
Comment (shu1513) [inline templates/register.html:1]: get rid of javascript from the html page.

**Review by shu1513 — COMMENTED** [review]
I changed a lot of things including hid 2 directories of binary files into .gitignore

**Pull request merged**